File: 71A - Way Too Long Words.py
def wayTooLongWords(inp):
    for i in range(inp):
        word = input()
        if len(word) >10:
            print(word[0]+str(len(word) - 2)+word[-1])
        else:
            print(word)

wayTooLongWords(int(input()))

File: 71A - Way Too Long Words.js
function wayTooLongWords(inp) {
  for (i = 0; i < inp; i++) {
    var word = readline();
    if (word.length > 10) {
      print(word[0] + (word.length - 2) + word[word.length - 1]);
    } else {
      print(word);
    }
  }
}

wayTooLongWords(parseInt(readline()));
